google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -35,7 +36,15 @@ module Bigquery
       # Google BigQuery. {Google::Cloud::Bigquery::Dataset} objects are created,
       # accessed, and deleted by Google::Cloud::Bigquery::Project.
       #
-      # See {Google::Cloud#bigquery}
+      # See {Google::Cloud#bigquery}.
+      #
+      # @attr_reader [String, nil] name The descriptive name of the project.
+      #   Only present if the project was retrieved with {#projects}.
```

Comment:
I think it's best to remove `Project#project_id` and just rely on the existing `Project#project` instead, do you agree?